File: Server/src/main/scala/Proxy.java
import akka.actor.ActorRef;
import net.MessageClass;

/**
 * @Author: xieshiqiang
 * @Date: 2018/12/7 16:32
 * @Description:
 **/
public class Proxy {
    private ActorRef sender;
    private MessageClass msg;
    private String session;
    private String originalMsg;

}

File: Server/src/main/scala/net/Proxy.scala
package net

import akka.actor.ActorRef

/**
  *
  * @Author: xieshiqiang 
  * @Date: 2018/12/7 16:10
  * @Description:
  **/
class Proxy {
  var sender:ActorRef
  var session:String
  var msg:MessageClass


  def init(sender:ActorRef, msg:String) = {
    this.sender = sender;
    this.
  }
}
